concepticondata/conceptlists/Mountain-1978-267.tsv

```diff
@@ -0,0 +1,268 @@
+Mountain-1978-267-97	97	mono aullador	howler monkey	106	433	HOWLER MONKEY
+Mountain-1978-267-98	98	marimonda, maquisapa	spider monkey	107	434	SPIDER MONKEY
+Mountain-1978-267-99	99	chigüiro, ronsoco	capybara	108	314	CAPYBARA
+Mountain-1978-267-100	100	oso hormiguero	anteater	109	181	ANTEATER
+Mountain-1978-267-101	101	paca, gua-gua, majas	paca	110	316	PACA
+Mountain-1978-267-102	102	caimán	crocodile	111	1857	CROCODILE
+Mountain-1978-267-103	103	iguana	iguana	112	1578	IGUANA
+Mountain-1978-267-104	104	tortuga	tortoise	113	895	TORTOISE
+Mountain-1978-267-105	105	saíno	collared peccary	115	435	COLLARED PECCARY
+Mountain-1978-267-106	106	agutí	agouti	117	313	AGOUTI
+Mountain-1978-267-107	107	rata	rat	118	1490	RAT
+Mountain-1978-267-108	108	gato	cat	119	1208	CAT
+Mountain-1978-267-109	109	cola	tail	121	1220	TAIL
+Mountain-1978-267-110	110	serpiente, culebra	snake	122	730	SNAKE
+Mountain-1978-267-111	111	anaconda	anaconda, water boa	123	896	ANACONDA (WATER BOA)
+Mountain-1978-267-112	112	sapo	toad	126	894	TOAD
+Mountain-1978-267-113	113	pájaro	bird	127	937	BIRD
+Mountain-1978-267-114	114	picaflor	hummingbird	128	1589	HUMMINGBIRD
+Mountain-1978-267-115	115	guacamayo	macaw	129	1630	MACAW
+Mountain-1978-267-116	116	tucán, pinsha	toucan	130	308	TOUCAN
+Mountain-1978-267-117	117	loro	parrot	131	882	PARROT
+Mountain-1978-267-118	118	perico, pihuicho	parakeet	132	886	PARAKEET
+Mountain-1978-267-119	119	gallinazo	buzzard	133	1865	BUZZARD
+Mountain-1978-267-120	120	pavo	curassow	134	315	CURASSOW
+Mountain-1978-267-121	121	lechuza	owl	135	735	OWL
+Mountain-1978-267-122	122	pava de monte	guan	136	437	GUAN
+Mountain-1978-267-123	123	gallina	hen	137	1514	HEN
+Mountain-1978-267-124	124	pescado	fish	138	227	FISH
+Mountain-1978-267-125	125	caribe	piranha	139	317	PIRANHA
+Mountain-1978-267-126	126	abeja	bee	140	665	BEE
+Mountain-1978-267-127	127	mosca	fly	141	1504	FLY (INSECT)
+Mountain-1978-267-128	128	pulga	flea	142	232	FLEA
+Mountain-1978-267-129	129	piojo	louse	143	1392	LOUSE
+Mountain-1978-267-130	130	zancudo	mosquito	144	1509	MOSQUITO
+Mountain-1978-267-131	131	comején, hormiga blanca	termite	145	883	TERMITE
+Mountain-1978-267-132	132	hormiga	ant	146	587	ANT
+Mountain-1978-267-133	133	araña	spider	147	843	SPIDER
+Mountain-1978-267-134	134	nigua pique	chigger	148	438	CHIGGOE (JIGGER FLEA)
+Mountain-1978-267-135	135	monte	bush	149	1683	BUSH
+Mountain-1978-267-136	136	pajonal, sabana	open grassland	150	260	SAVANNA
+Mountain-1978-267-137	137	cerro, loma	hill	151	614	HILL
+Mountain-1978-267-138	138	árbol	tree	152	906	TREE
+Mountain-1978-267-139	139	hoja	leaf	153	628	LEAF
+Mountain-1978-267-140	140	flor	flower	155	239	FLOWER
+Mountain-1978-267-141	141	fruta	fruit	156	1507	FRUIT
+Mountain-1978-267-142	142	savia	sap	157	345	SAP
+Mountain-1978-267-143	143	raíz	root	158	670	ROOT
+Mountain-1978-267-144	144	hierba	grass	161	606	GRASS
+Mountain-1978-267-145	145	maíz	corn, maize	162	506	MAIZE
+Mountain-1978-267-146	146	yuca	manioc	163	927	MANIOC
```

Comment:
Interestingly, different lists with 'manioc' are mapped either to MANIOC and CASSAVA. In Huber it's linked to CASSAVA.

Reply:
I am not sure whether I got the difference between MANIOC and CASSAVA right. Is manioc the plant and cassava already a product of that plant or are they considered different plant species?

Reply:
As far as I can tell, CASSAVA refers to the eadible component of the concept while MANIOC - to the plant species.

Reply:
The distinction makes sense, but then I am still not sure which one is more applicable here. Probably MANIOC (the plant), as we have yuca and harina de yuca?